### DISCUSSION TITLE
Clarification Please

### DISCUSSION THREAD

**Discussion by sandstone42** [GrandOrgue/grandorgue#907] [Help] [answered]
The help file defines Pipe999LoadRelease as
(boolean, default: reverse of percussive setting) If true, the release part is loaded from the first attack sample.
To me, this is confusing because "first attack sample" is not clearly defined. What is the "first attack sample?" An example would really help.

**Comment by larspalo** [comment, marked as answer]
Well, the attack samples can contain releases too, but they don't have to. In many newer samplesets the releases are cut off from the attacks and only loaded separately. In old times, the (mostly only single) samples were complete with both loop and cue markers and nothing else was expected for a Pipe999.

If you specify Pipe999LoadRelease=N, then the release from that sample file will not be loaded (from that attack). It's of course still possible to define that same file as a release later, but that's overthinking things, in my opinion... If Pipe999LoadRelease=Y (the default for Percussive=N, and thus really not necessary to define) then a release cue is expected in the sample file and that release will be loaded (from the attack file - but as a release that you don't need to specify as any extra release for that pipe!).

Main reason for using Pipe999LoadRelease at all is if the attack samples doesn't include a cue point indicating the release. Then you must specify Pipe999LoadRelease=N to let GO know that it should not bother looking for a release in that file.

The help line of defaulting to reverse of percussive might be cryptic, but think of it like so: if Percussive=N then LoadRelease=Y, if Percussive=Y then LoadRelease=N. In essence, a percussive file is played one shot from beginning to end without any loops or cues and actually cannot have any releases at all. It's only the case above where the attack doesn't include any cue and release that you'll have: Percussive=N and LoadRelease=N. It's not really useful (or possible without warnings at least) to have Percussive=Y and LoadRelease=Y as explained above.

I hope that was an understandable explanation.

**Reply by larspalo** [reply]
And first attack sample is the attack specified in Pipe999=
